src/core/exec-invoke.c

```diff
@@ -4638,6 +4663,8 @@ int exec_invoke(
         int ngids = 0, ngids_after_pam = 0;
         int socket_fd = -EBADF, named_iofds[3] = EBADF_TRIPLET;
         size_t n_storage_fds, n_socket_fds, n_extra_fds;
+        _cleanup_close_pair_ int token_fds[2] = EBADF_PAIR;
+        _cleanup_(pidref_done_sigkill_wait) PidRef bpftoken_pid = PIDREF_NULL;
```

Comment:
We don't do anything with bpf token, so these aren't appropriate variable names to use.